moteur_server_rest/routes.py

```diff
@@ -74,7 +74,7 @@ def handle_status(workflow_id):
     workflow_status = "RUNNING" if pids else "UNKNOWN"
 
     if workflow_status != "RUNNING":
-        workflow_out_path = os.path.join(document_root, workflow_id, "workflow.out")
+        workflow_out_path = os.path.join(document_root, workflow_id, "process.out")
```

Comment:
process.out should have very few logs, the one not handled by logback.
I understand the log we'll search for should be logged normally by logback and be in workflow.out.